website/src/components/footer/footer.astro

```diff
@@ -112,7 +112,10 @@ const { class: className } = Astro.props;
           target="_blank"
           rel="noopener noreferrer"
           class="header-github-link"
```

Comment:
ah i think this was meant to show the same icon as in the header

Reply:
I'll make that change